recipes/sagemath-db-graphs/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = "20161026" %}
+{% set name = "sagemath-db-graphs" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.bz2
+  url: http://files.sagemath.org/spkg/upstream/graphs/graphs-{{ version }}.tar.bz2
+  sha256: 622de9ad45106177db4f61334c2dde88ade560ce087d3b37bc2edae4292d4b28
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
The software that uses this is not available on Windows? (Just curious, b/c packaging data like this would be quite simple to ship for Windows too.)